string collision1
string collision2
float64[] depth
geometry_msgs/Point[] position
geometry_msgs/Point[] normal
geometry_msgs/Point[] force1
geometry_msgs/Point[] force2
int32 num